_posts/2020-09-14-release-0-21-5.md

```diff
@@ -0,0 +1,87 @@
+to the backend of MathOptInterface, such as specialized dictionaries which act
+like arrays at first, [but switch to dictionaries when elements are deleted](https://github.com/jump-dev/MathOptInterface.jl/pull/1142).
+
+[@blegat](https://github.com/blegat) also implemented an option that changes the
+way to compute which bridges to use in a transformation, opening the way for us
+to experiment with different bridging schemes in the future.
+
+On the solver front, the conic solver SCS in particular received a
+[lot](https://github.com/jump-dev/SCS.jl/pull/182)
+[of](https://github.com/jump-dev/SCS.jl/pull/185)
+[attention](https://github.com/jump-dev/SCS.jl/pull/189)
+[on](https://github.com/jump-dev/SCS.jl/pull/191)
+[improving](https://github.com/jump-dev/SCS.jl/pull/192)
+[performance](https://github.com/jump-dev/SCS.jl/pull/196). This led to very
+large gains formulating large problems in our benchmarks. In the worst case, a
+problem that never finished loading into the solver [now takes 10 seconds](https://github.com/jump-dev/SCS.jl/issues/181).
+
+We also have a number of contributors to thank.
+* [@ilancoulon](https://github.com/ilancoulon) took a [deep dive into the bridging system](https://github.com/jump-dev/MathOptInterface.jl/pull/1101)
+* [@rtwalker](https://github.com/rtwalker) implemented a new bridge to [convert
+between `ZeroOne` and `Integer` constraints](https://github.com/jump-dev/MathOptInterface.jl/pull/1099)
+* [@dourouc05](https://github.com/dourouc05) completed a multi-year odyssey
+to [expose irreducible infeasible subsystems (IIS)](https://github.com/jump-dev/JuMP.jl/pull/2300).
+He also made a number of [other](https://github.com/jump-dev/MathOptInterface.jl/pull/1149)
+[fixes](https://github.com/jump-dev/JuMP.jl/pull/2317)
+and [documentation](https://github.com/jump-dev/JuMP.jl/pull/2316)
+[improvements](https://github.com/jump-dev/JuMP.jl/pull/2264).
+* [@yashcodes](https://github.com/yashcodes) made some [helpful](https://github.com/jump-dev/JuMP.jl/pull/2222)
+[improvements](https://github.com/jump-dev/JuMP.jl/pull/2203) to JuMP's error
+messages and the [documentation in MathOptInterface](https://github.com/jump-dev/MathOptInterface.jl/pull/1050).
+* [@egbuck](https://github.com/egbuck) [fixed a bug](https://github.com/jump-dev/MathOptInterface.jl/pull/1124)
+in objective support for various file formats.
+* [@andrewrosemberg](https://github/com/andrewrosemberg) [fixed a bug](https://github.com/jump-dev/JuMP.jl/pull/2319)
+that affected heuristic callbacks.
+
+We also thank testing improvments from [@iainnz](https://github/com/iainnz),
+documentation improvements from  [@zlatanvasovic](https://github/com/zlatanvasovic),
+[@mtanneau](https://github/com/mtanneau) and [@aks1996](https://github/com/aks1996).
+
+For a full list of the merged pull requests and closed issues that contributed
+to this release, see the tag notes for [JuMP](https://github.com/JuliaOpt/JuMP.jl/releases/tag/v0.21.4)
```

Comment:
This is maybe weird that here, it is the release note of `v0.21.4` but indeed most of the changes are there